pkg/restapi/operation/models.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package operation
+
+type createKeystoreReq struct {
+	// Sequence is a counter for the keystore configuration to ensure that clients are properly synchronized.
+	Sequence int `json:"sequence"`
+	// Controller is an entity that is in control of the keystore.
+	Controller string `json:"controller"`
+}
+
+type createKeyReq struct {
+	KeystoreID string `json:"keystoreID"`
+	KeyType    string `json:"keyType"`
```

Comment:
do we also need to declare the key's purpose?

cc @Baha-sk

Reply:
I think that we will definitely need more properties there (at least "controller" or list of controllers for the entities who controls the key). I just started with those 2 as they are needed right now. This struct will be updated in progress.